iscsi/initiator.go

```diff
@@ -3,6 +3,7 @@ package iscsi
 import (
 	"bufio"
 	"fmt"
+	"github.com/sirupsen/logrus"
```

Comment:
Needs to be put in the section after "time" since it's a third party dependency. E.g.
```
"time"

"github.com/sirupen/logus"

"github.com/longhorn/go-iscsi-helper/util"
```